tests/test_text_onnx_embeddings.py

```diff
@@ -47,6 +47,9 @@
     "jinaai/jina-embeddings-v2-base-de": np.array(
         [-0.0085, 0.0417, 0.0342, 0.0309, -0.0149]
     ),
+    "jinaai/jina-embeddings-v2-base-code": np.array(
+        [0.0145, -0.0164,  0.0136, -0.0170,  0.0734]
+    ),
```

Comment:
was it produced with:
```
from sentence_transformers import SentenceTransformer

model = SentenceTransformer('jinaai/jina-embeddings-v2-base-code', trust_remote_code=True)
s.encode(['hello world'])[:,:5]
```

(it gives me a different result: 
```
array([[ 0.20987998, -0.2358965 ,  0.19645616, -0.24518925,  1.0565469 ]],
      dtype=float32)
```

Reply:
I remember filling these values from ST for "Hello". Then after https://github.com/qdrant/fastembed/actions/runs/9962765584/job/27527279039?pr=301, I realized it was the wrong str, I got the values from there.

Reply:
`normalize_embeddings=True` made it.